cpp/benchmarks/stream_compaction/unique.cpp

```diff
@@ -77,3 +80,47 @@ NVBENCH_BENCH_TYPES(nvbench_unique, NVBENCH_TYPE_AXES(data_type, keep_option))
   .set_name("unique")
   .set_type_axes_names({"Type", "KeepOption"})
   .add_int64_axis("NumRows", {10'000, 100'000, 1'000'000, 10'000'000});
+
+template <typename Type, cudf::duplicate_keep_option Keep>
+void nvbench_unique_list(nvbench::state& state, nvbench::type_list<Type, nvbench::enum_type<Keep>>)
+{
+  if constexpr (Keep != cudf::duplicate_keep_option::KEEP_FIRST) {
+    state.skip("Skip unwanted benchmarks.");
```

Comment:
Out of curiosity, why do we exclude `KEEP_FIRST` here?

Reply:
From what I understood from reading the implementation, `KEEP_ANY` and `KEEP_FIRST` do the same thing for `unique`

Reply:
Ah, you are right.
https://github.com/rapidsai/cudf/blob/dec8bde1d5cc7462e52535a4e26f2c1be507a237/cpp/src/stream_compaction/unique.cu#L57-L58
Probably worth adding a comment explaining the reason of skipping `KEEP_FIRST`.

Reply:
I copied this from the primitive types benchmark, but we are skipping everything *but* `KEEP_FIRST`. I should probably modify both the benchmarks and skip `KEEP_ANY`

Reply:
Works for me. Thanks!